### PR TITLE
fix: Bad code fence indenting in ordered lists

### PR DESCRIPTION
**Pull request by nschonni**
### Why:

Split out commit from #541 for markdown sections that cause parser issues with markdownlint because of incorrect indenting of start/end code fences

### What's being changed:

- Un-started table row. Caused Crowdin to mangle it in the translated copies
- Code fences inside of ordered list that didn't have at lease 3 spaces and some that had different open/closing indents

### Check off the following:
- [x] All of the tests are passing.
- [x] I have reviewed my changes in staging.
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Comment by janiceilene** [comment]
Thanks for opening a PR @nschonni! I'll get this triaged for review 🌟

**Comment by nschonni** [comment]
@janiceilene I've removed the content changes from the other PR now, since the original issue requiring them has been fixed in Markdownlint. They are still content issues, so I'll leave the for review here

File: content/admin/installation/installing-github-enterprise-server-on-google-cloud-platform.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Before launching {% data variables.product.product_location %} on Google Cloud P
 {% data variables.product.prodname_ghe_server %} is supported on the following Google Compute Engine (GCE) machine types. For more information, see [the Google Cloud Platform machine types article](https://cloud.google.com/compute/docs/machine-types).
 
 | High-memory   |
-  ------------- |
+| ------------- |
```
Comment (nschonni) [inline content/admin/installation/installing-github-enterprise-server-on-google-cloud-platform.md:30]: This is the only functional change

**Comment by nschonni** [comment]
@janiceilene @rachmari looks like the stale bot doesn't want to dismiss the label after pushing/rebasing

**Comment by janiceilene** [comment]
Looking at this again briefly, this can be reviewed by any writer! I removed @rachmari and I as reviewers and I'll move this to the `anyone` column for triage.

File: content/admin/enterprise-management/configuring-high-availability-replication-for-a-cluster.md [hunk 2]
```diff
@@ -294,19 +304,27 @@ You can monitor the progress on any node in the cluster, using command-line tool
...
-      ghe-storage replication-status
+  ```
+  ghe-storage replication-status
+  ```
 
 - Monitor replication of Pages data:
 
-      ghe-dpages replication-status
+  ```
+    ghe-dpages replication-status
```
Comment (martin389) [inline content/admin/enterprise-management/configuring-high-availability-replication-for-a-cluster.md:326]:
```suggestion
  ghe-dpages replication-status
```

File: content/admin/installation/installing-github-enterprise-server-on-google-cloud-platform.md [hunk 3]
```diff
@@ -63,18 +63,18 @@ Based on your user license count, we recommend these machine types.
...
-  ```
-  This table identifies the required ports and what each port is used for.
-
-  {% data reusables.enterprise_installation.necessary_ports %}
+    ```shell
+    $ gcloud compute firewall-rules create <em>RULE-NAME</em> \
+    --network <em>NETWORK-NAME</em> \
+    --allow tcp:22,tcp:25,tcp:80,tcp:122,udp:161,tcp:443,udp:1194,tcp:8080,tcp:8443,tcp:9418,icmp
+    ```
+    This table identifies the required ports and what each port is used for.
```
Comment (martin389) [inline content/admin/installation/installing-github-enterprise-server-on-google-cloud-platform.md:75]: Looks like this table isn't rendering as expected in stage:
https://docs-675--code-fence-indents.herokuapp.com/en/enterprise-server@2.22/admin/installation/installing-github-enterprise-server-on-google-cloud-platform#configuring-the-firewall

Here's what it looks like for me:
image

For comparison with prod:
https://docs.github.com/en/enterprise-server@2.22/admin/installation/installing-github-enterprise-server-on-google-cloud-platform#configuring-the-firewall

**Review by martin389 — COMMENTED** [review]
Thanks for this PR @nschonni! I've added a couple of suggestions as comments.

File: content/admin/installation/installing-github-enterprise-server-on-google-cloud-platform.md (hunk 3 above)
Comment (nschonni) [reply content/admin/installation/installing-github-enterprise-server-on-google-cloud-platform.md]: OK, went back to 3 spaces and it seems happy again.
I'm finding the markdown process a little frustrating, as it doesn't seem to follow CommonMark or GFM conventions

Comment (martin389) [reply content/admin/installation/installing-github-enterprise-server-on-google-cloud-platform.md]: Thanks for the fix! 🎉

**Comment by janiceilene** [comment]
👋 @martin389 It looks like this one's ready for your 👀 again

**Comment by martin389** [comment]
Hi @nschonni 👋 

I had a look at [Configuring high availability replication for a cluster](https://docs-675--code-fence-indents.herokuapp.com/en/enterprise-server@2.22/admin/enterprise-management/configuring-high-availability-replication-for-a-cluster), and some of the rendering might need to be checked:

  - In step 9, of [Adding passive nodes to the cluster configuration file](https://docs-675--code-fence-indents.herokuapp.com/en/enterprise-server@2.22/admin/enterprise-management/configuring-high-availability-replication-for-a-cluster#adding-passive-nodes-to-the-cluster-configuration-file), the code block for the third bullet point is indented too far left:
image

  - In step 12 of [Adding passive nodes to the cluster configuration file](https://docs-675--code-fence-indents.herokuapp.com/en/enterprise-server@2.22/admin/enterprise-management/configuring-high-availability-replication-for-a-cluster#adding-passive-nodes-to-the-cluster-configuration-file), the contents of the warning admonition seem to have escaped and are rendering underneath the block:
image
This has also reset the procedure's numbering back to 1, when it should be continuing to step 13 from there.

**Review by martin389 — CHANGES_REQUESTED** [review]
Thanks for these changes, @nschonni! 🎉 

I've finished my review, and added a comment above for one of the sections:
- [Configuring high availability replication for a cluster](https://docs-675--code-fence-indents.herokuapp.com/en/enterprise-server@2.22/admin/enterprise-management/configuring-high-availability-replication-for-a-cluster) - some small tweaks needed.

The rest looks good to me: 🚀 
- [Installing GitHub Enterprise Server on Google Cloud Platform](https://docs-675--code-fence-indents.herokuapp.com/en/enterprise-server@2.22/admin/installation/installing-github-enterprise-server-on-google-cloud-platform) 
- [Creating a pre-receive hook environment](https://docs-675--code-fence-indents.herokuapp.com/en/enterprise-server@2.22/admin/policies/creating-a-pre-receive-hook-environment)
[Creating a pre-receive hook script](https://docs-675--code-fence-indents.herokuapp.com/en/enterprise-server@2.22/admin/policies/creating-a-pre-receive-hook-script)
- [Setting your username in Git](https://docs-675--code-fence-indents.herokuapp.com/en/free-pro-team@latest/github/using-git/setting-your-username-in-git)

**Comment by nschonni** [comment]
🤦 sorry about the repeat reviews. I rebased and checked that file on dev side-by-side again with poduction and they're looking the same.
I notice that there appears to be something funny going on with the list numbering with the warning/note blocks resetting the numbering, but it appears in production as well. It looks like the correct `<ol start='#'>` is being correctly set, but the styling is going back to 1 🤷

**Comment by martin389** [comment]
> 🤦 sorry about the repeat reviews. I rebased and checked that file on dev side-by-side again with poduction and they're looking the same.
> I notice that there appears to be something funny going on with the list numbering with the warning/note blocks resetting the numbering, but it appears in production as well. It looks like the correct `<ol start='#'>` is being correctly set, but the styling is going back to 1 🤷

No worries, thanks for checking on this 👍  It does seem like it should all be rendering as expected, so I'll do some digging around here too.

**Comment by martin389** [comment]
👋 For docs tooling folks:

The procedure numbering resets a few times in this staging article: ["Configuring high availability replication for a cluster"](https://docs-675--code-fence-indents.herokuapp.com/en/enterprise-server@2.22/admin/enterprise-management/configuring-high-availability-replication-for-a-cluster#adding-passive-nodes-to-the-cluster-configuration-file). Specific examples are listed below:

- "Assigning active nodes to the primary datacenter": [(stage)](https://docs-675--code-fence-indents.herokuapp.com/en/enterprise-server@2.22/admin/enterprise-management/configuring-high-availability-replication-for-a-cluster#assigning-active-nodes-to-the-primary-datacenter) [(prod)](https://docs.github.com/en/enterprise-server@2.22/admin/enterprise-management/configuring-high-availability-replication-for-a-cluster#assigning-active-nodes-to-the-primary-datacenter) - Step 5 in prod becomes step 1 in stage."
- "Adding passive nodes to the cluster configuration file": [(stage)](https://docs-675--code-fence-indents.herokuapp.com/en/enterprise-server@2.22/admin/enterprise-management/configuring-high-availability-replication-for-a-cluster#adding-passive-nodes-to-the-cluster-configuration-file) [(prod)](https://docs.github.com/en/enterprise-server@2.22/admin/enterprise-management/configuring-high-availability-replication-for-a-cluster#adding-passive-nodes-to-the-cluster-configuration-file) - Step 2 in prod becomes step 1 in stage. Numbering also resets again a few more times throughout this section.

**Comment by nschonni** [comment]
Sounds like it might have been an internal issue that was fixed https://github.com/github/docs/issues/1552#issuecomment-732401111
I'll try rebasing now to see if it fixes it

**Comment by nschonni** [comment]
@martin389 rebase looks like it fixed it 😄

**Review by martin389 — APPROVED** [review]
Re-reviewed after tooling fixes were added in https://github.com/github/docs/issues/1552#issuecomment-732401111:

- [Configuring high availability replication for a cluster](https://docs-675--code-fence-indents.herokuapp.com/en/enterprise-server@2.22/admin/enterprise-management/configuring-high-availability-replication-for-a-cluster) - 👍 
- [Installing GitHub Enterprise Server on Google Cloud Platform](https://docs-675--code-fence-indents.herokuapp.com/en/enterprise-server@2.22/admin/installation/installing-github-enterprise-server-on-google-cloud-platform) - 👍 
- [Creating a pre-receive hook environment](https://docs-675--code-fence-indents.herokuapp.com/en/enterprise-server@2.22/admin/policies/creating-a-pre-receive-hook-environment) - 👍 
- [Creating a pre-receive hook script](https://docs-675--code-fence-indents.herokuapp.com/en/enterprise-server@2.22/admin/policies/creating-a-pre-receive-hook-script) - 👍 
- [Setting your username in Git](https://docs-675--code-fence-indents.herokuapp.com/en/free-pro-team@latest/github/using-git/setting-your-username-in-git) - 👍

**Comment by martin389** [comment]
Thanks again for these fixes, @nschonni! 🎉

File: content/admin/installation/installing-github-enterprise-server-on-google-cloud-platform.md (hunk 1 above)
Comment (nschonni) [reply content/admin/installation/installing-github-enterprise-server-on-google-cloud-platform.md:30]: Looks like this got Crowdin to parse correctly again from the changes in #1603

**Pull request merged**